config/Dockerfiles/release-notes.py

```diff
@@ -63,18 +63,15 @@ def get_remaining_changes(pulls):
     changes = []
     # iterate over pulls backward so that list can be removed in place
     for i in xrange(len(pulls) - 1, -1, -1):
-        skip = False
         labels = pulls[i].get_labels()
         for label in labels:
             if label.name == "release":
-                skip = True
+                continue
```

Comment:
won't this skip the inner loop?  Also, will the two lines below this ever run?

Reply:
I see this is outdated, you can ignore it

Reply:
I just changed while you were commenting :)